src/resources/Sources/Sources.ts

```diff
@@ -83,7 +83,7 @@ export default class Sources extends Resource {
 
     duplicate(sourceId: string, newSourceName: string, logicalIndex?: string) {
         return this.api.post<SourceModel>(
-            this.buildPath(`${Sources.baseUrl}/${sourceId}/duplicate`, {newSourceName, logicalIndex})
+            this.buildPath(`${Sources.baseUrl}/${sourceId}/duplicate`, {newSourceName, logicalIndexId: logicalIndex})

```

Comment:
Just rename the parameter of the `duplicate` function, it's not a breaking change to do so because (afaik), in JS, the parameter of a function can not be mapped by name, only by order.